tagstudio/src/qt/platform_strings.py

```diff
@@ -6,11 +6,13 @@
 
 import platform
 
+from .translations import Translations
+
 
 class PlatformStrings:
-    open_file_str: str = "Open in file explorer"
+    open_file_str: str = Translations["file.open_file.other"]
 
     if platform.system() == "Windows":
-        open_file_str = "Open in Explorer"
+        open_file_str = Translations["file.open_file.windows"]
     elif platform.system() == "Darwin":
-        open_file_str = "Reveal in Finder"
+        open_file_str = Translations["file.open_file.macos"]
```

Comment:
These should use the `file.open_location.x` keys. The variable name `open_file_str` is a bit misleading since it's supposed to refer to opening file locations rather than files themselves.